### PR TITLE
MathMagicians - Feature/components

### PR DESCRIPTION
**Pull request by clintonjosephs**
Hello 😊,

<div><br/>via <a href="https://giphy.com/theoffice/">The Office</a> on <a href="https://giphy.com/gifs/theoffice-MNmyTin5qt5LSXirxd">GIPHY</a></div>

I have successfully achieved task two for the MathMagician project 😄. Please see below the things accomplished in this PR
- Delete all the boilerplate from CRA (text, images, styles)
- Create a directory called components.
- Inside components, created a new Calculator.js file.
- In Calculator.js, create a React component that matches the wireframe design

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -1,12 +1,12 @@
...
 import ReactDOM from 'react-dom';
 import './index.css';
-import App from './App';
+import Calculator from './components/Calculator';
 import reportWebVitals from './reportWebVitals';
 
 ReactDOM.render(
   <React.StrictMode>
-    <App />
+    <Calculator />
```
Comment (rukundoeric) [inline src/index.js]: - [x] <b>REQUIRED:</b> Project requires that you import `App` component here not `Calculator`.  `Calculator` component must be imported in the App component instead. Kindly do that to comply with project requerements.

**Review by rukundoeric — CHANGES_REQUESTED** [review]
Hi @clintonjosephs, 

_Great job so far 👏 💯 👍. However, there are few issues we need to address before we move on. Please, check the comments below._
## Highlights
  - Follows git workflow.  💯
  - Codebase looks clean and professional. 💯 

## Required Changes ♻️
  - [ ] Check the comments under the review.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
  - [ ] Check the comments under the review.

Cheers and Happy coding!👏👏👏

__Feel free to leave any questions or comments in the PR thread if something is not 100% clear.__

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
> As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: src/index.js (hunk 1 above)
Comment (clintonjosephs) [reply src/index.js]: Thank you for pointing this out, I have effected the changes as recommended

**Review by julie-ify — APPROVED** [review]
## Hi @clintonjosephs,

# Highlights
- Good job implementing the required changes from the previous reviewer ✔️ 

# Status: Approved ✔️ 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 

# Congratulations! 🎉 

Approved emoji

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**